engine/controllers/command_line_parser.cc

```diff
@@ -457,3 +456,104 @@ void CommandLineParser::EngineGet(CLI::App* parent) {
         [engine_name] { commands::EngineGetCmd().Exec(engine_name); });
   }
 }
+
+void CommandLineParser::ModelUpdate(CLI::App* parent) {
+  auto model_update_cmd =
+      parent->add_subcommand("update", "Update configuration of a model");
+  model_update_cmd->group(kSubcommands);
+  model_update_cmd->add_option("--model_id", cml_data_.model_id, "Model ID")
+      ->required();
+
+  // Add options dynamically
+  model_update_cmd->add_option("--name", model_update_options_.name, "Name");
```

Comment:
@vansangpfiev @namchuai @nguyenhoangthuan99 I know this is C++, but I wonder whether there is a way to automatically generate this sort of repetitive code from an array of param names. 

- `command_line_parser.cc`
- `model_config.h`
- `model_upd_cmd.h`

That said, with AI, I think repetitive coding can be handled via LLM.

Reply:
Suggestion: pass it through Jan/endpoint with a refactor prompt 🙏

Reply:
Got it, I refactored it to make it more clear